docs/index.md

```diff
@@ -1,3 +1,42 @@
 # Home
 
-Welcome to Toon Deobfuscated, your central hub for documentation on Eneco Toon thermostats, also known as [xx] and [yy] in other countries.
+Welcome to Toon Deobfuscated, your central hub for documentation and tooling
+for the Eneco Toon thermostats, also known as Engie Electrabel Boxx and Viesgo
+Toon in other countries. This device was originally developed by Quby, a Dutch
+company specializing in smart home technology. They ultimately got acquired by
+Eneco.
+
+This documentation is aimed at providing comprehensive information about the
+Toon thermostat's internal workings. The core audience includes developers,
+engineers, and enthusiasts interested in understanding and potentially
+contributing to the community-run ecosystem surrounding the Toon thermostat.
+
+## Motivation
+The official Toon thermostat platform has proven to be a useful tool for
+managing home heating and energy consumption.
+
+In terms of hardware, even the oldest models are still very capable,
+offering features like Z-Wave, Thermostat interface and a large touchscreen,
+and a processor that can run a Linux distribution. However, older models no
+longer receive updates, and official support has been discontinued.
+
+In terms of repurposing, there is a great community of developers still
+working on extending the functionality of the Toon thermostat, even to this
+day.
+
+However, as time goes by, the community tools have gone stale, links no longer
+always work, and documentation is outdated. It also depends on Eneco / Quby's
+official infrastructure, which will eventually be shut down.
+
+## Goals
+The goal of this documentation is to provide a comprehensive and up-to-date
+resource for anyone seeking to repurpose the Toon thermostat. This
+documentation is compiled from various sources, including community
+contributions, official documentation, and reverse engineering efforts.
+
+## Early stages
+
+This documentation is currently in its early stages. We are actively working
+on expanding the content to cover various aspects of the Toon thermostat.
+
+Initially, this documentation will only focus on the now-defunct Toon v1.
```

Comment:
there's some v2 bits in here, and the OS is not -that- much different. Maybe s/only/mostly/?